src/pages/settings-resources/forms/ReportLayoutsForm.js

```diff
@@ -0,0 +1,198 @@
+import { Box, Grid, Typography } from '@mui/material'
+import Table from 'src/components/Shared/Table'
+import FormShell from 'src/components/Shared/FormShell'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { useContext, useEffect, useState } from 'react'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import toast from 'react-hot-toast'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { ControlContext } from 'src/providers/ControlContext'
+import { useForm } from 'src/hooks/form'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+
+const ReportLayoutsForm = ({ labels, maxAccess, row, window: w }) => {
```

Comment:
leave the window as window

Reply:
i cant , i want to use window.open also in other method